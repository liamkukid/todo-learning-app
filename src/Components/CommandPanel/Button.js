import PropTypes from 'prop-types';
import style from './button.module.scss';

export default function Button({ title, onClick }) {
  return (
    <button
      className={style.button}
      type="button"
      onClick={onClick}
      value={title}
    >
      {title}
    </button>
  );
}
Button.propTypes = {
  title: PropTypes.string.isRequired,
  onClick: PropTypes.func.isRequired,
};
