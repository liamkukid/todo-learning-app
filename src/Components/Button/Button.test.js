import renderer from 'react-test-renderer';
import Button from './Button';

it('renders corretly for Button', () => {
  const tree = renderer.create(<Button title="name" onClick={() => {}} />);
  expect(tree).toMatchSnapshot();
});
