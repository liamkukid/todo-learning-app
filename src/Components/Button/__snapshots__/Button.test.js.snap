// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders corretly for Button 1`] = `
<button
  className="button"
  onClick={[Function]}
  type="button"
  value="name"
>
  name
</button>
`;
